extensions/markdown-language-features/src/features/preview.ts

```diff
@@ -439,6 +439,9 @@ class MarkdownPreview extends Disposable implements WebviewResourceProvider {
 		if (hrefPath[0] !== '/') {
 			// Fix #93691, use this.resource.fsPath instead of this.resource.path
 			hrefPath = path.join(path.dirname(this.resource.fsPath), hrefPath);
+		} else {
+			// Handle any normalized file paths
+			hrefPath = hrefPath.replace('file///', '');
```

Comment:
After investigation, I noticed that this issue was caused inadvertently by https://github.com/microsoft/vscode/pull/114553/files where the links were normalised to display images. This fix handles both absolute links and file:// links

Reply:
Thanks for the suggestion. I've made the change The latest update now handles `file:/` links as well. I've tested this change both on Windows and Mac for multiple paths.

Windows:
![Windows](https://user-images.githubusercontent.com/3755450/108286283-b74d9680-7180-11eb-8e30-c7f6e33131b1.JPG)

Mac:
<img width="1440" alt="Mac" src="https://user-images.githubusercontent.com/3755450/108285918-11019100-7180-11eb-9a75-dc16b6815c90.png">

From what I can see, `file://` paths aren't supported right now. I can't seem to build a version of vscode prior to https://github.com/microsoft/vscode/pull/114553/files so not sure if they were supported before then